portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
```

Comment:
@inodb will the custom css be another file or do we have to overwrite one of the files in source control? We already have a global property for overwriting the portal's main css file to change the look and feel, pointing to a custom css file. See code

```java
    String global_style = GlobalProperties.getProperty("global_css");
    String special_style = GlobalProperties.getProperty("special_css");
```
in `css_include_standard.jsp` and `css_include.jsp`.